planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/interface.cpp

```diff
@@ -265,6 +265,15 @@ bool LaneChangeInterface::canTransitFailureState()
     return true;
   }
 
+  if (module_type_->is_near_terminal()) {
+    log_debug_throttled("Unsafe, but ego is approaching terminal. Continue lane change");
+
+    if (module_type_->isRequiredStop(post_process_safety_status_.is_trailing_object)) {
+      log_debug_throttled("Module require stopping");
+    }
```

Comment:
This condition is only to add a log message, is this okay ?

Reply:
Yes, this is okay.
We will address the stopping method in the redesign, so for now just adding log is fine.